addons/mail/static/src/core/common/attachment_view.xml

```diff
@@ -3,7 +3,12 @@
 
     <t t-name="mail.AttachmentView">
         <div t-if="state.thread.attachmentsInWebClientView.length > 0" class="o-mail-Attachment">
-            <div class="o_attachment_control popout d-print-none" t-on-click="onClickPopout"><i class="fa fa-window-restore" aria-hidden="Pop out" title="Pop out"/></div>
+            <div class="o_attachment_control popout d-print-none cursor-pointer"
+                 t-on-click="onClickPopout"
+                 data-tooltip="Pop the preview in a separate window for dual screens."
```

Comment:
I think the wording is a bit weird. Maybe something simpler like this?
```suggestion
                 data-tooltip="Open preview in a separate window."
```

Reply:
"Pop the preview" and "for dual screens" is just incorrect English 🤷

Reply:
Odooglish at it again :laughing:

Reply:
Oopsie doopsie, well, I'll change that then, thanks you :smile: